checks/rabbitmq/aliveness_check_test.go

```diff
@@ -0,0 +1,23 @@
+package rabbitmq
+
+import (
+	"github.com/hellofresh/health-go/checks/http"
```

Comment:
use `goimport` formatting - std imports must be on top and separated by emoty line from other imports